docs/quickstarts/nextjs.mdx

```diff
@@ -1,35 +1,26 @@
 ---
-title: Next.js Quickstart
-description: Add authentication and user management to your Next.js app with Clerk.
+title: Next.js Quickstart (App Router)
+description: Add authentication and user management to your Next.js app.
+search: false
+metadata:
+    robots:
+        index: false
```

Comment:
It seems strange to me that this page would be excluded from our search and search engines. I just wanted to check that this was deliberate?

Reply:
No, not intentional! This was left over from the previous approach where we had keyless as a separate path. Good catch. Fixed in #1978